org.eclipse.lemminx/src/test/java/org/eclipse/lemminx/services/XMLFormatterTest.java

```diff
@@ -284,22 +284,22 @@ public void rangeSelectWithinText() throws BadLocationException {
 	@Test
 	public void testProlog() throws BadLocationException {
 		String content = "<?xml version=   \"1.0\"       encoding=\"UTF-8\"  ?>" + lineSeparator();
-		String expected = "<?xml version=\"1.0\" encoding=\"UTF-8\" ?>" + lineSeparator();
+		String expected = "<?xml version=\"1.0\" encoding=\"UTF-8\"?>" + lineSeparator();

```

Comment:
Remplace lineSeparator with "\r\n". The idea is to use lineSepatrator when it is required (ex : when XML doesnt define lien separator in the XML content and after formatting a line separator is generated. In this case lien separator is used.